pkg/service/labels/constants.go

```diff
@@ -0,0 +1,21 @@
+package labels
+
+const (
+	NameLabel = "Name" //Capital N for Aws
+
+	CreatorLabel        = "creator"
+	SpawnerServiceLabel = "spawner-service"
```

Comment:
Just call it "spawner" and drop the "-service"?

Reply:
updated.